casepro/contacts/models.py

```diff
@@ -245,12 +256,12 @@ def as_json(self, full=True):
         """
         Prepares a contact for JSON serialization
         """
-        result = {'uuid': self.uuid, 'name': self.name}
+        result = {'id': self.pk, 'name': self.get_display_name()}
```

Comment:
Is it still possible for them to see the full backend uuid? Wasn't this something they rolled over before?

Just wondering whether there's value there for them cross referencing with RapidPro